From: Andreas Moog <amoog@ubuntu.com> 
Last-Update: 2013-01-04
Description: gcc_46
 gcc4.5 and later are more restrictive with c++ function calls,
 this trivial patch fixes build failures with -fpermissive
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3
--- a/src/njamedit.cpp
+++ b/src/njamedit.cpp
@@ -114,7 +114,7 @@ void NjamEngine::LevelEditor()
 				key = SDLK_a;
 			else if (CheckForSave())
 			{
-				LogFile::LogFile("Saving maps");
+				LogFile("Saving maps");
 				m_Maps.Save(filename);
 				level_type_was = level_type;
 				changed = false;
@@ -131,7 +131,7 @@ void NjamEngine::LevelEditor()
 					char types[2][5] = { "COOP", "DUEL" };
 					sprintf(buf, "levels/%s.%s\0", filename, types[level_type]);
 					sprintf(filename, "%s\0", buf);
-					LogFile::LogFile("Saving maps");
+					LogFile("Saving maps");
 					m_Maps.Save(filename);
 					level_type_was = level_type;
 					changed = false;
